meeteval/der/__main__.py

```diff
@@ -24,7 +26,20 @@ def md_eval_22(
 def cli():
     from meeteval.wer.__main__ import CLI
 
-    cli = CLI()
+    class DerCLI(CLI):
+        def add_argument(self, command_parser, name, p):
+            if name == 'exclude_overlap':
+                command_parser.add_argument(
+                    '-1', '--exclude-overlap',
+                    action='store_true',
+                    help='Limits scoring to single-speaker regions. '
```

Comment:
@TCord Could you add a sentence, where the user should be careful? Or was that only relevant, if you want to calculate the overlap only DER?